internal/cmd/beta/network-area/route/describe/describe.go

```diff
@@ -137,6 +138,19 @@ func outputResult(p *print.Printer, outputFormat string, route *iaas.Route) erro
 		table.AddRow("Prefix", *route.Prefix)
 		table.AddSeparator()
 		table.AddRow("Nexthop", *route.Nexthop)
+		table.AddSeparator()
+		var sb strings.Builder
+		count := 0
+		for _, v := range *route.GetLabels() {
+			count++
+			if value, ok := v.(string); ok {
+				sb.WriteString(value)
+				if len(*route.GetLabels()) != count {
+					sb.WriteString(" ")
+				}
+			}
+		}
+		table.AddRow("Labels", sb.String())
```

Comment:
Here I think we could avoid the strings.Builder and make it something like this. I think it's simpler to read and also more inline with other commands


```go
if route.Labels != nil && len(*route.Labels) > 0 {
    labels := []string{}
    for key, value := range *route.Labels {
        labels = append(labels, fmt.Sprintf("%s: %s", key, value))
    }

    table.AddSeparator()
    table.AddRow("Labels", strings.Join(labels, "\n"))
}
```

Reply:
Also, this has two other benefits:
- Only shows the Labels row if there is at least a label
- Shows both key and value of each label, one per row